### PR TITLE
Extend README with examples how to reproduce

### PR DESCRIPTION
**Pull request by pat-s**
Contributes to https://github.com/ropenscilabs/tic/issues/58

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,5 +2,138 @@
...
+4. Run `usethis::use_ci()` to set up all requirements needed for the CI integration of your package.
+
+## Way 2: Starting from scratch locally
+
+This is the detailed way going step by step from the bottom up.
+Looking here can be useful if you already have an R package and just want to add some functionality to it, e.g. using `roxygen` with markdown (point 3) or enabling CI for your package (point 6).
+
+1. Use `usethis::create_package("<path-to-package-dir")`. Here is an example with a package called `tic.package` stored within the `~/git/` directory:
+
+```r
```
Comment (krlmlr) [inline README.md]: Can you please indent these blocks?

File: README.md [hunk 2]
```diff
@@ -2,5 +2,138 @@
...
+
+1. Use `usethis::create_from_github()` within RStudio to automatically create a fork of this repo as a new RStudio project. You may need to set up your SSH credentials first. See [this guide](http://happygitwithr.com/ssh-keys.html) if you're having problems. Its definitely worth getting this function running as it saves you so much time in the future! Alternatively, fork this repo on Github and then create a new RProject within RStudio (File -> New Project -> Version Control -> Github). 
+2. Change the DESCRIPTION file to your needs (package title, maintainer, links).
+3. Rename the `.Rproj` file and the folder of the project to the name of your package.
+4. Run `usethis::use_ci()` to set up all requirements needed for the CI integration of your package.
+
+## Way 2: Starting from scratch locally
+
+This is the detailed way going step by step from the bottom up.
+Looking here can be useful if you already have an R package and just want to add some functionality to it, e.g. using `roxygen` with markdown (point 3) or enabling CI for your package (point 6).
```
Comment (krlmlr) [inline README.md]: Please check numbering.

**Review by krlmlr — COMMENTED** [review]
Thanks. So this means that even when forking we need a full run of `use_ci()` ? This simplifies matters -- we can just have a section that explains why `use_ci()` is important regardless if you fork or set up from scratch, and then have shorter sections on forking or recreating. Probably the same is true for the other example projects.

File: tic.R [hunk 3]
```diff
@@ -13,3 +13,6 @@ if (Sys.getenv("id_rsa") != "") {
     add_step(step_build_pkgdown()) %>%
     add_step(step_push_deploy())
 }
+
+get_stage("after_success") %>%
```
Comment (krlmlr) [inline tic.R]: This is added by `add_package_checks()` already.

**Comment by pat-s** [comment]
> So this means that even when forking we need a full run of use_ci() ?

From the 5 steps that `use_ci()` runs:

- travis_enable()
- use_template()
- use_tic()
- use_travis_deploy()
- travis_set_pat()

We would essentially only need `travis_enable()` and `use_travis_deploy()`. But why confuse the reader arguing why we only need a subset of `use_ci()` rather than just telling "just use `use_ci()`?

#### Disable forking at all?

Even though the forking way seems shorter, we should think about if we want that: The created repo will always be labeled as a "fork" of the example repo. 
This is probably not desired by the user and might also lead to confusion if the package that is created from the example repo becomes more popular -> "why is it a fork of this example repo?".

I would even raise the question if we should make the example repos unforkable and only provide a "short" and "long version" how to recreate this state in the README (the "from scratch way").

Further we could think of adding a function that wraps around all the functions that I listed as "option 2".

It would be nice to be able to just call a function named `tic::create_new_project()` that takes as arguments

- project_type: by default "package"
- local dir incl. package name
- roxygen_md (TRUE/FALSE)
- license
- ci (default travis + appveyor)

We can also discuss this in an issue rather than in this PR. But let me know what you think about this (before I continue here) :) @krlmlr

**Comment by pat-s** [comment]
@krlmlr Can you take a look here? We should have decided this before the CRAN release of *tic*.

**Comment by krlmlr** [comment]
Thanks! We can't forbid forking but we should definitely outline that forking is not the way to start a package, only to play. I'll tweak the README.

**Pull request merged**